Error: Cannot find module 'fp-ts/boolean'

Importing from node.js (not running Typescript) using the import:
`import keycloak from 'fastify-keycloak-adapter';`

Throws error: 
`Error: Cannot find module 'fp-ts/boolean'`

--- a/package.json
+++ b/package.json
@@ -34,13 +34,13 @@
     "fastify-cookie": "^5.6.0",
     "fastify-jwt": "^4.1.3",
     "fastify-plugin": "^3.0.1",
+    "fp-ts": "^2.11.10",
     "grant": "^5.4.20",
     "wildcard-match": "^5.1.2"
   },
   "devDependencies": {
     "@types/jest": "^27.4.1",
     "@types/node": "^16.11.26",
-    "fp-ts": "^2.11.8",
     "jest": "^27.5.1",
     "prettier": "^2.5.1",
     "ts-jest": "^27.1.3",

